CMakeLists.txt

```diff
@@ -1057,7 +1059,7 @@ if (UA_ENABLE_DISCOVERY_MULTICAST)
     target_compile_definitions(open62541 PRIVATE -DMDNSD_DYNAMIC_LINKING_EXPORT)
 endif()
 # Generate properly versioned shared library links on Linux
-SET_TARGET_PROPERTIES(open62541 PROPERTIES SOVERSION 0 VERSION "${OPEN62541_VER_MAJOR}.${OPEN62541_VER_MINOR}.${OPEN62541_VER_PATCH}")
+SET_TARGET_PROPERTIES(open62541 PROPERTIES SOVERSION "${SONAME}" VERSION "${OPEN62541_VER_MAJOR}.${OPEN62541_VER_MINOR}.${OPEN62541_VER_PATCH}")
```

Comment:
Can't we use `${OPEN62541_VER_MAJOR}` directly in here, instead of creating an additional `SONAME` variable?

Reply:
> Can't we use `${OPEN62541_VER_MAJOR}` directly in here, instead of creating an additional `SONAME` variable?

yes, of course, we can do it...

BTW: 
I think we should agree to change `${OPEN62541_VER_MAJOR}` whenever the ABI and/or API changes to be sure having a valid `SONAME` , it's the #2923 issue